docs/core/diagnostics/built-in-metrics-aspnetcore.md

```diff
@@ -146,7 +154,7 @@ Available staring in: ASP.NET Core 8.0
 
 | Name     | Instrument Type | Unit (UCUM) | Description    |
 | -------- | --------------- | ----------- | -------------- |
-| `aspnetcore.rate_limiting.queued_requests` | UpDownCounter | `{request}` | Number of requests that are currently queued, waiting to acquire a rate limiting lease. |
+| `aspnetcore.rate_limiting.queued_requests` | UpDownCounter | `{request}` | Number of requests that are currently queued waiting to acquire a rate limiting lease. |
```

Comment:
@tdykstra  I think this reads better without the `,`

Reply:
I think it needs the comma.

Reply:
I like the comma too.